dev/scripts/evergreen-kubernetes-setup.sh

```diff
@@ -1,14 +1,25 @@
 #!/usr/bin/env bash
 
+if [ -z "$3" ]; then
+  echo 'Usage: ./dev/scripts/evergreen-kubernetes-setup.sh BIN_PATH DOCKER_REPO DOCKER_NAME' > /dev/stderr
+  exit 1
+fi
+
+bin=$1
+docker_repo=$2
+docker_name=$3
```

Comment:
A comment explaining which docker repository and name wouldn't hurt.

Reply:
Sorry, what do you mean by "which"?

Reply:
I just mean that in when you run this script you have to give it a docker repository and an image name and that it might be nice to explain what those are or where one might get them.

Reply:
Just realized how stupid this is. The script doesn't care about any docker repo, all it does is load an image into kind. It should just accept the name of a Docker image. Changed now!